src/reference/00-Getting-Started/07B-Appending-Values.md

```diff
@@ -0,0 +1,84 @@
+---
+out: Appending-Values.html
+---
+
+Appending values
+----------------
+
+### Appending to previous values: `+=` and `++=`
+
+Assignment with `:=` is the simplest transformation, but keys have other
+methods as well. If the `T` in `SettingKey[T]` is a sequence, i.e. the key's
+value type is a sequence, you can append to the sequence rather than
+replacing it.
+
+- `+=` will append a single element to the sequence.
+- `++=` will concatenate another sequence.
+
+For example, the key `sourceDirectories in Compile` has a `Seq[File]` as its
```

Comment:
When we discuss the build.sbt DSL, the nomenclature could be something like:

```
organization :=   { name.value }
-----------  --    -------------
key        operator    body
--------------------------------
  setting (or task) expression
```

But colloquially we often say "organization is a setting."

In this case, `sourceDirectories in Compile` is a scoped key of type `SettingKey[Seq[File]]`.